src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -100,6 +98,18 @@
     @Rule
     public final SystemOutRule systemOut = new SystemOutRule().enableLog().mute();
 
+    private final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
+            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted", null, null,
+            getClass(), null);
+
+    private final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
+            Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditFinished", null, null,
+            getClass(), null);
+
+    private final LocalizedMessage errorCounterMessage = new LocalizedMessage(0,
```

Comment:
please name it "errorCounterOneMessage" to clearly define that is not general message, but parametrized by "1"